tests/query-node/run-tests.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+set -e
+
+SCRIPT_PATH="$(dirname "${BASH_SOURCE[0]}")"
+cd $SCRIPT_PATH
+mkdir -p hydra-test/
+cd hydra-test
+
+npx @dzlzv/hydra-cli scaffold --wsProviderUrl=ws://joystream-node:9944/ --projectName=Test
+
+cp ../../../types/augment/all/defs.json ./mappings/typedefs.json
+export TYPES_JSON=../../mappings/typedefs.json
+
+# Joystream mappings and graphsql schema
+# cp ../../../query-node/mappings/* ./mappings/
+# cp ../../../query-node/schema.graphql ./
+
+npx @dzlzv/hydra-cli codegen
+
+# Setup the postgres database
+docker-compose up $1 -d db
+
+function cleanup() {
+    docker-compose down -v
+}
+
+trap cleanup EXIT
+
+yarn db:migrate
+
+# Bring up remaining services
+docker-compose up $1 -d
```

Comment:
We have to be sure that `hydra-indexer` image is always fresh. One option is that  I can add `:latest` tag to the compose file template, so, assuming the latest image is built at the previous step, ensures it is fresh. Or perhaps simply clean all the images afterwards?

Reply:
I had the same thought, which is why I added the `$1` there so you can pass an optional `build` argument when calling the script, depending on whether you just want to run services again without rebuilding.